### ISSUE TITLE
Kubeflow missing on ML search

### ISSUE THREAD

**Issue by ronaldpetty** [cncf/landscape2#403]
Hi,

TLDR - ML/AI is hot, can we search the landscape by that quality?

Prose -

I am unable to filter by machine learning. Specifically was targeting Kubeflow.

https://landscape.cncf.io/card-mode?category=scheduling-orchestration&project=incubating,sandbox,member

Returns Kubeflow, but if you filter by industry "Machine Learning" Kubeflow is not listed (nor anything else). 

https://landscape.cncf.io/card-mode?category=scheduling-orchestration&project=incubating,sandbox,member&industries=machine-learning

I am not certain if "Industry" really means industry, or does it mean "Industry" and other qualitative features/labels. Maybe the "Category" should add ML/AI but that is just a thought. 

There is another ticket that may related to this #361 . 

Love the landscape! Fwiw, I am trying to help the ai whitepaper for the cncf and it would be helpful if we can focus readers to the CNCF projects (which will change over time) that have some relation to ML/AI. Else, we have to pick winners and we may be wrong or stale.

**Comment by tegioz** [comment]
Closed by #619
